cmd/support_test.go

```diff
@@ -1,35 +0,0 @@
-package cmd
-
-import (
```

Comment:
For some reason this test does not work in our windows workflow after our update.
I tried to check in my windows machine and this test works. Suggesting some strange issue with the worker windows version.

Reply:
I added the test to the yaml.

Reply:
Why add to yaml? That reduces test coverage.

Reply:
This is the most basic of tests, so we can just reimplement if it is wrong.

Reply:
For example, look at how you are implementing the tests for color. You are taking a different approach than what we see in this test. Maybe use the same approach.